openfe/utils/visualization.py

```diff
@@ -89,7 +91,8 @@ def _draw_molecules(
     atoms_list: Collection[Set[int]],
     bonds_list: Collection[Set[int]],
     atom_colors: Collection[Dict[Any, Tuple[float, float, float, float]]],
-    highlight_color: Tuple[float, float, float, float]
+    highlight_color: Tuple[float, float, float, float],
+    atom_mapping: Dict[Tuple[int, int], Dict[int, int]] = None,
```

Comment:
```suggestion
    atom_mapping: Optional[Dict[Tuple[int, int], Dict[int, int]]] = None,
```

looks like mypy isn't complaining about this, but let's explicitly allow `None` here.

Reply:
uh that is a nice way of typing that!